mikidown/mikiwindow.py

```diff
@@ -1264,7 +1264,15 @@ def closeEvent(self, event):
                           top-level widgets
             saveState: Restores the state of this mainwindow's toolbars
                        and dockwidgets
+            BUG: here cos we want a quit or minimize to tray and a popup
+                 we dont wanna crash the application with out warning and WTF atmo
+                 this is dark stuff.. across xpatform btw so help useful.. said pedro..
+                 We also want to expose the settings in a simple way....
+                 Am not comfartable with the Qt.ini.. stuff
```

Comment:
The reason why I chose the Qt Ini stuff is because:

* No adding additional dependencies just for configuration
* The config file should be editable with a text editor in case things go wrong
* This is as crossplatform as it gets when working without additional deps

That being said, python does have its own ConfigParser library, but that doesn't escape the not being comfortable with ini files that you mentioned before.

That being said, *.ini files are as simple as one could get were it not for how Qt handles setting lists.

Could you elaborate on what makes the *.ini files hard to edit both by hand and with the GUI config?